app/models/manageiq/providers/azure/network_manager/floating_ip.rb

```diff
@@ -1,2 +1,7 @@
 class ManageIQ::Providers::Azure::NetworkManager::FloatingIp < ::FloatingIp
+  belongs_to :load_balancer
+
+  def self.available
+    where(:vm_id => nil, :load_balancer_id => nil)
```

Comment:
right, seems like only LB and VM can be associated here, so this should be good for now